sys/fido2/ctap/ctap.c

```diff
@@ -1792,12 +1793,15 @@ static int _ctap_decrypt_rk(ctap_resident_key_t *rk, ctap_cred_id_t *id)
 
 static int _write_state_to_flash(const ctap_state_t *state)
 {
-    return fido2_ctap_mem_write(state, CTAP_FLASH_STATE_PAGE, 0, CTAP_FLASH_STATE_SZ);
+    return fido2_ctap_mem_write(state,
+                                fido2_ctap_mem_get_state_flashpage_number(), 0,
+                                CTAP_FLASH_STATE_SZ);
 }
 
 static int _read_state_from_flash(ctap_state_t *state)
 {
-    return fido2_ctap_mem_read(state, CTAP_FLASH_STATE_PAGE, 0, sizeof(*state));
+    return fido2_ctap_mem_read(state, fido2_ctap_mem_get_state_flashpage_number(), 0,
+                               sizeof(*state));
 }
 
 static int _write_rk_to_flash(const ctap_resident_key_t *rk, int page, int offset)
```

Comment:
This indirection is not needed, is it? It seems to me that you want to have `_save_rk` here (the only user of this function). But I would keep the name `_write_rk_to_flash` to be comparable to `_write_state_to_flash`.